Assignment8/src/Ex8/Refer.java

```diff
@@ -0,0 +1,66 @@
+package Ex8;
```

Comment:
Please maintain separate file for establishing connection . Change the java file name from Refer to some other meaningful name